share/lib/python/neuron/rxd/geometry3d/ctng.pyx

```diff
@@ -9,6 +9,11 @@ cimport cython
 from neuron.rxd.rxdException import RxDException
 import neuron
 
+
+"""
+The ctgn module
```

Comment:
ctng?

Reply:
This is a great addition to the binary installer suite and is extremely useful. Obvously a lot of work. Many thanks!   I'd like to understand many of the issues that were resolved in this implementation in greater depth and think I need an interactive tour of the pull request from you so I can ask questions. Lets setup a zoom or skype meeting. Maybe others would like to join as well.

Reply:
> ctng?

We had to give a name to the module, due to some Cython issue. Without any idea of this module stands for, I used the same as the name.

>  I need an interactive tour of the pull request from you so I can ask questions

For sure. But I will need to prepare myself since a lot of small decisions were taken in the way, some of them some months ago.

Reply:
Why are the wheelhouse binaries part of the repository? Don't they get built everytime the repository gets a commit?

Reply:
those were there just part of convenience. building part we will add inn spearate PR.

Reply:
I just deleted them from the whole branch